Add outline functionality (#5)
WayScience/IDR_stream: README.md

## Example Usage

Example usage of `idrstream` can be found at [example.ipynb](example.ipynb).

> This notebook was run as a python script ([example.py](example.py)) and hence has no output for the final cell.

Reviewer: ```suggestion
We ran this notebook as a python script ([example.py](example.py)) and hence the notebook has no output for the final cell.
```

Active (instead of passive voice)

> **Note**: `idrstream` can be used to extract object oulines as extra metadata by passing `extra_metadata=["object_outlines"]` during `idrstream.run_stream()`. Similarly, the desired batch numbers can be chosen with `batch_nums=[#,#,#]`.

Reviewer: ```suggestion
**Note**: You can use `idrstream` o extract object oulines as extra metadata by passing `extra_metadata=["object_outlines"]` during `idrstream.run_stream()`.
Similarly, you can choose the desired batch numbers with `batch_nums=[#,#,#]`.
```

Shifting to active voice (no passive voice, please!). I also split the multi-sentence line to two lines.

Author: Thank you!
